deploy/olm-catalog/compliance-operator/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -167,6 +167,11 @@ metadata:
     support: Red Hat Inc.
   name: compliance-operator.v0.1.47
   namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.arm64: supported
```

Comment:
We'd love to have compliance operator for ARM too, but currently it's not being built (downstream) so it MUST NOT be included here.

Reply:
Oh good catch, this should be corrected.

Reply:
OK, Dropped arm64.